sbi/inference/snpe/snpe_base.py

```diff
@@ -271,6 +271,8 @@ def train(
 
             # Train for a single epoch.
             self._neural_net.train()
+            log_prob_sum = 0
+            e_start = time.time()
```

Comment:
could you rename this to `epoch_start_time`?

Reply:
resolved with 200cd5a